src/modules/accessChecks.ts

```diff
@@ -348,7 +348,10 @@ export const accessChecks = {
             return notAllowed(tr('No access to hire stream or section type'));
         },
 
-        update: (session: Session, section: SectionWithInterviewRelation): AccessCheckResult => {
+        update: (
+            session: Session,
+            section: SectionWithInterviewRelation & { interviewers: User[] },
```

Comment:
Maybe `{ interviewers: User[] }` could be added to type `SectionWithInterviewRelation` itself?